samtranslator/swagger/swagger.py

```diff
@@ -673,6 +675,9 @@ def set_path_default_apikey_required(self, path: str) -> None:
 
             security = existing_non_apikey_security + apikey_security
 
+            if method_name == "options" and not AddApiKeyRequiredToCorsPreflight:
+                security = existing_non_apikey_security
```

Comment:
What's the difference with this and the change in `add_auth_to_method`?

Reply:
This matters if they define AddApiKeyRequiredToCorsPreflight in the globals instead of at the individual Api level.

Reply:
Why?